Model/Resolver/setRzpPaymentDetailsOnCart.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Razorpay\Magento\Model\Resolver;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\QuoteGraphQl\Model\Cart\CheckCartCheckoutAllowance;
+use Magento\QuoteGraphQl\Model\Cart\GetCartForUser;
+use Magento\QuoteGraphQl\Model\Cart\SetPaymentMethodOnCart as SetPaymentMethodOnCartModel;
+use Razorpay\Magento\Model\PaymentMethod;
+use Razorpay\Magento\Model\Config;
+
+/**
+ * Mutation resolver for setting payment method for shopping cart
+ */
+class SetRzpPaymentDetailsOnCart implements ResolverInterface
```

Comment:
@enishant , this need to be renamed `SetRzpPaymentDetailsForOrder`

Reply:
@ramth05 Renamed resolver name SetRzpPaymentDetailsOnCart to setRzpPaymentDetailsForOrder